packages/block-library/src/post-comments-form/index.php

```diff
@@ -28,8 +28,15 @@ function render_block_core_post_comments_form( $attributes, $content, $block ) {
 	$form               = ob_get_clean();
 	$wrapper_attributes = get_block_wrapper_attributes( array( 'class' => $classes ) );
 
-	// Enqueue the comment-reply script.
-	wp_enqueue_script( 'comment-reply' );
+	
+	/**
+	 * Enqueue the comment-reply script.
+	 * Line commented until we fix this behaviour causing a display issue.
```

Comment:
Is there an open GH issue about this problem? It would be great to have it linked to this PR. 🙂

Reply:
I have to create it. Is not so easy to explain 😆

Reply:
(I've reverted this change for now, see updated PR description.)